src/test/scala/biowdl/test/TestPipelineSuccess.scala

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright (c) 2018 Biowdl
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy of
+ * this software and associated documentation files (the "Software"), to deal in
+ * the Software without restriction, including without limitation the rights to
+ * use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of
+ * the Software, and to permit persons to whom the Software is furnished to do so,
+ * subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS
+ * FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR
+ * COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER
+ * IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN
+ * CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+ */
+
+package biowdl.test
+
+import java.io.File
+
+import nl.biopet.utils.biowdl.PipelineSuccess
+
+trait TestPipelineSuccess extends TestPipeline with PipelineSuccess {}
```

Comment:
Maybe it would be good to add an actual test to see if the structs are being handled correctly.

Reply:
What do you mean? The pipeline is the actual test. Right?

Reply:
The pipeline could finish 'successfully' without handling the structs correctly. If something goes wrong with the structs, it might end up with, for example, an empty readgroups list even though there are supposed to be readgroups. It would still finish with exit status 0, but it wouldn't be correct.

Reply:
Oh wait, this is already tested by the testing utils, nevermind...